care/fixtures/fixtures.md

```
@@ -0,0 +1,254 @@
+# CARE Fixture System
+
+The fixture system provides a consistent, API-driven way to seed development and test data. All data is created through Django REST Framework's `APIClient`, ensuring viewset side-effects (slug generation, auto-created records, validations) are exercised — just like real API calls.
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)